src/lib/bookingApiParameters.ts

```diff
@@ -2,7 +2,7 @@ import type { BusStop } from './busStop';
 import type { Capacities } from './capacities';
 import type { Coordinates } from './location';
 
-export type BookingRequestParameters = {
+export type CheckBookingValidityParameters = {
```

Comment:
```suggestion
export type WhiteListRequest = {
```

Man kann dann `BlackListRequest` als Alias von Whitelist Request definieren.

Terminologie für Anfragen = `...Request`
Terminologie für Antworten = `...Response` (ggf. hier wieder Typalias `BlacklistResponse` = `WhitelistResponse` falls zutreffend